iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/FittingOrgWordsMatchVoter.java

```diff
@@ -121,11 +121,21 @@ private String filterName(String name) {
             filteredName = StringUtils.remove(filteredName, charToFilter);
         }
         
-        if (minWordLength > 0) {
-            filteredName = StringUtils.removePattern(filteredName, "\\b\\w{1," + minWordLength + "}\\b");
+        if (wordToRemoveMaxLength > 0) {
+            filteredName = StringUtils.removePattern(filteredName, "\\b\\w{1," + wordToRemoveMaxLength + "}\\b");
             filteredName = filteredName.trim().replaceAll(" +", " ");
         }
         
         return filteredName;
     }
+
+    //------------------------ toString --------------------------
+    
+    @Override
+    public String toString() {
+        return "FittingOrgWordsMatchVoter [charsToFilter=" + charsToFilter
```

Comment:
You may consider using either guava's `MoreObjects.toStringHelper()` or `ToStringBuilder` from `commons-lang`.